### PR TITLE
Deploy staging server continuously with github actions

### PR DESCRIPTION
**Pull request by kurtwheeler**
## Issue Number

#174 

## Purpose/Implementation Notes

This is most of the way there. The one issue I cannot figure out is how to refresh the `http_auth` for the elasticsearch client. Once it expires our search breaks. As far as I can tell, the elasticsearch libraries only accept a string for that setting and don't provide a way to refresh it, so I've open an issue with them about it: https://github.com/elastic/elasticsearch-dsl-py/issues/1377.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

https://github.com/AlexsLemonade/resources-portal/runs/786004899?check_suite_focus=true worked!!!

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kurtwheeler** [comment]
Could this be related? https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/issues/108

File: infrastructure/database.tf [hunk 1]
```diff
@@ -24,25 +24,25 @@ resource "aws_db_instance" "postgres_db" {
...
   engine_version = "9.6.11"
   auto_minor_version_upgrade = false
-  instance_class = "db.${var.database_instance_type}"
+  instance_class = var.database_instance_type
   name = "resources_portal"
-  port = "${var.database_port}"
-  username = "${var.database_user}"
-  password = "${var.database_password}"
+  port = "5432"
+  username = "rppostgresuser"
```
Comment (davidsmejia) [inline infrastructure/database.tf:30]: 🤔

**Comment by davidsmejia** [comment]
Still looking this over but on stage we arent serving static files.. i see media in urls.py but no mention of static

Also the link in config/common.py should probably be https://docs.djangoproject.com/en/3.0/howto/static-files/deployment/

File: infrastructure/database.tf (hunk 1 above)
Comment (kurtwheeler) [reply infrastructure/database.tf:30]: ?

**Comment by kurtwheeler** [comment]
Ok @davidsmejia I believe I am done with this PR!

**Comment by kurtwheeler** [comment]
Ok I've added those env vars to github actions!

**Comment by kurtwheeler** [comment]
Hmm, you know, I'm not entirely 100% sure. We might not need subnets?

**Pull request merged**